src/nodes/fmod_event_emitter_3d.h

```diff
@@ -11,18 +11,22 @@ namespace godot {
         GDCLASS(FmodEventEmitter3D, Node3D)
 
     private:
+        Vector3 _velocity;
         void set_space_attribute_impl(const Ref<FmodEvent>& p_event) const;
         void free_impl();
 
     public:
-        FmodEventEmitter3D() = default;
+        FmodEventEmitter3D() : _velocity(0, 0, 0) {}
         ~FmodEventEmitter3D() override = default;
 
         virtual void _ready() override;
         virtual  void _process(double delta) override;
         void _notification(int p_what);
         virtual void _exit_tree() override;
 
+        void set_velocity(const Vector3& velocity);
```

Comment:
This can be added to FmodEventEmitter template by adding a vector template param to it.  
This way it will be implemented for both 2D and 3D emiters.